api/dataservices/interface.go

```diff
@@ -14,9 +14,10 @@ import (
 type (
 	// DataStore defines the interface to manage the data
 	DataStore interface {
-		Open() (newStore bool, err error)
+		Open() error
 		Init() error
 		Close() error
+		IsNew() bool
```

Comment:
please no. we want isNew _not_ to be a property of the store - its an extremely temporary state that essentially only needs to be known to the initial creation. (yes, I did quite a bit of work to get rid of it in the refactor)

Reply:
Ok, I thought it was added in the forked branch by you. I will remove it.